### PR TITLE
Storage and Documentation v1.4

### PR DESCRIPTION
**Pull request by eugenefdw**
- Storage:
  - Default storage locations for userdata and lesson data has been split and changed: e8ccdf6 
  - Changed file format for lesson storage to use a separate line dictating column type, instead of user-unfriendly escape characters.
    - There is now another row above field names, with three possible values: Tested, Not Tested, and Hint. The parser actually only checks for case-insensitive T, N and H. Any other values will result in the lesson not loading.
    - Question and Answers are now purely internal working terminology. As Quiz tests in both directions, the first two TESTED columns will be used, instead of specific column marking.

- Documentation:
   - Removes references to AB4 in AboutUs
   - Added back navigation bar

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -287,34 +287,75 @@ These are questions that are labelled difficult by the user.
...
+image::CsvLessonStorageSequenceDiagram.png[width="800"]
 
 Lessons are parsed by CsvLessonListStorage, being converted between Lesson and List<String[]> formats. The List<String[]> format is the primary format handled by CsvUtil for reading and writing to CSV files.
 
-For lessons, there are three main components when saved to a CSV file.
+For lessons, there are three main sections when saved to a CSV file.
 
 ===== Name
-Lesson names are equivalent to their filenames.
+Lesson names are equivalent to their file names.
```
Comment (DigiPie) [inline docs/DeveloperGuide.adoc]: Suggestion for clarity:

```suggestion
A lesson's name in BrainTrain is equivalent to its file name.
```

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -287,34 +287,75 @@ These are questions that are labelled difficult by the user.
...
-
-Core fields start with `CORE_ESCAPE` character, `\*`. When a field without `*` is found, all subsequent fields are assumed to be optional.
-
-Question and answer fields have the `QUESTION_ESCAPE` character, `?`, and `ANSWER_ESCAPE` character, `@`, respectively after the core character, `*`.
+The header is represented in the file as the first two lines. Each line represents a specific set of data, as outlined below.
+
+The first line represents the type of the corresponding column. Each column can have three types:
+
+. *Tested* +
+This marks the column as a core value. The first two instances of Tested in the file will be used in quizzes, and all remaining Tested values are treated as Not Tested.
```
Comment (DigiPie) [inline docs/DeveloperGuide.adoc]:
```suggestion
This marks the column as a core value. The first two instances of *Tested* in the file will be used in quizzes, and all remaining Tested values are treated as *Not Tested*.
```

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -995,74 +1047,29 @@ remember it better
...
 *MSS*
 
-1. User downloads file from a source (e.g Internet, other users).
-2. User uses import command.
-3. App loads the file as a lesson into the app shows the content of the lesson.
-4. App requests confirmation.
-5. App saves lesson to user profile.
+. User downloads file from a source (e.g Internet, other users).
+. User moves the file to the lessons folder.
+. User enters `reloadLessons` command.
```
Comment (DigiPie) [inline docs/DeveloperGuide.adoc]:
```suggestion
. User enters `reload` command.
```

File: src/test/data/CsvLessonListStorageTest/emptyHeaderValue/test.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+﻿TESTED,TESTED,
```
Comment (DigiPie) [inline src/test/data/CsvLessonListStorageTest/emptyHeaderValue/test.csv]:
```suggestion
﻿TESTED,TESTED
```

Comment (eugenefdw) [reply src/test/data/CsvLessonListStorageTest/emptyHeaderValue/test.csv]: As intended as part of the test

**Pull request merged**